docs/docs/validating-plugin-options.md

```diff
@@ -0,0 +1,230 @@
+---
+title: Validating plugin options
+---
+
+To help users [configure plugins](/docs/configuring-usage-with-plugin-options/) correctly, a plugin can optionally define a schema to enforce a type for each option. When users use a plugin, Gatsby will validate that the provided options match the schema.
+
+## How to define an options schema
+
+To define their options schema, plugins export a [`pluginOptionsSchema`](/docs/node-apis/#pluginOptionsSchema) from their `gatsby-node.js`. This function gets passed an instance of [Joi](https://joi.dev) to define the schema with.
+
+For example, consider the following plugin called `gatsby-plugin-console-log`:
```

Comment:
This is the same example used throughout the "Configuring usage with plugin options" doc, which is right before this one, however I figured it wouldn't hurt to re-introduce it since folks might open this page directly.

Reply:
Agreed! Good thinking 👏